casper/src/main/scala/coop/rchain/casper/blocks/proposer/Proposer.scala

```diff
@@ -164,9 +165,7 @@ object Proposer {
         // TODO: temp this is done after validation
 //        MultiParentCasper.handleValidBlock(b) >>
         // broadcast hash to peers
-        CommUtil[F].sendBlockHash(b.blockHash, b.sender) >>
-        // Publish event
-        EventPublisher[F].publish(MultiParentCasper.createdEvent(b))
```

Comment:
As mentioned in [commit message](https://github.com/rchain/rchain/pull/3758/commits/5ed7d33970ab05bd1254b45d617fb67d102c811d) the logic should be correctly/fully implemented. Maybe it makes sense to add TODO comment or create a corresponding issue?

Reply:
We already have issue tracking improvements to event system.
- #3585